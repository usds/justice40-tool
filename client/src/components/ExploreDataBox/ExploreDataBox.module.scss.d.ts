declare namespace ExploreDataBoxNamespace {
    export interface IExploreDataBoxScss {
      summaryBoxContainer: string;
      fileDownIcon: string;
    }
  }

declare const ExploreDataBoxScssModule: ExploreDataBoxNamespace.IExploreDataBoxScss & {
    /** WARNING: Only available when "css-loader" is used without "style-loader" or "mini-css-extract-plugin" */
    locals: ExploreDataBoxNamespace.IExploreDataBoxScss;
  };

  export = ExploreDataBoxScssModule;
