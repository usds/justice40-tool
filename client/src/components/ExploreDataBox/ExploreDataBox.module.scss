@use '../../styles/design-system.scss' as *;

.summaryBoxContainer{

    @include u-margin-top(2.5);

    .fileDownIcon{
        margin-bottom: -4px;
        @include u-padding-left(1);
    }
}
