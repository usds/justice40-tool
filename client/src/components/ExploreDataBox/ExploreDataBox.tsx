import React from 'react';
import {SummaryBox, SummaryBoxContent, SummaryBoxHeading} from '@trussworks/react-uswds';
import {useIntl} from 'gatsby-plugin-intl';

import * as EXPLORE_COPY from '../../data/copy/explore';

import * as styles from './ExploreDataBox.module.scss';

// @ts-ignore
import fileDownIcon from '/node_modules/uswds/dist/img/usa-icons/file_download.svg';


export interface IExploreDataBoxProps {}

const ExploreDataBox = ({}: IExploreDataBoxProps) => {
  const intl = useIntl();

  return (
    <SummaryBox className={styles.summaryBoxContainer}>

      <SummaryBoxHeading headingLevel='h2'>
        {intl.formatMessage(EXPLORE_COPY.EXPLORE_DATA_BOX.TITLE)}
        <img tabIndex={0} className={styles.fileDownIcon} src={fileDownIcon}
          alt={intl.formatMessage(EXPLORE_COPY.SIDE_PANEL_INIT_STATE_ICON_ALT_TEXT.PLUS)}
        />
      </SummaryBoxHeading>

      <SummaryBoxContent>
        {EXPLORE_COPY.EXPLORE_DATA_BOX_BODY}
      </SummaryBoxContent>

    </SummaryBox>
  );
};

export default ExploreDataBox;
