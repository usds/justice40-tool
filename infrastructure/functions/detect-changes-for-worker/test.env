REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1