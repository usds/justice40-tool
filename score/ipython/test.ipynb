{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
