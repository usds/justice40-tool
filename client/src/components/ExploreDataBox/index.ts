import ExploreDataBox from './ExploreDataBox';
export default ExploreDataBox;
