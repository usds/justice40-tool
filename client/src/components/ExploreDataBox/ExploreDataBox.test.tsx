import React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../test/testHelpers';
import ExploreDataBox from './ExploreDataBox';

describe('rendering of ExploreDataBox Component', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <ExploreDataBox />
      </LocalizedComponent>,
  );
  it('checks if component renders', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
