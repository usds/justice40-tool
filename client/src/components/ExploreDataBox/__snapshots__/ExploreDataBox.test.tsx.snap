// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of ExploreDataBox Component checks if component renders 1`] = `
<DocumentFragment>
  <div
    class="usa-summary-box"
    data-testid="summary-box"
  >
    <div
      class="usa-summary-box__body"
    >
      <h2
        class="usa-summary-box__heading"
      >
        Get the data
        <img
          alt="
      a plus icon indicating that the user can zoom in
    *"
          src="test-file-stub"
          tabindex="0"
        />
      </h2>
      <div
        class="usa-summary-box__text"
      >
        
    Download the data with documentation and shapefile from the 
        <a
          class="usa-link"
          href="/en/downloads"
        >
          downloads
        </a>
         page.
  
      </div>
    </div>
  </div>
</DocumentFragment>
`;
